administrator/components/com_associations/views/associations/tmpl/side-by-side.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die; 
+
+JHtml::_('behavior.formvalidator');
+JHtml::_('behavior.keepalive');
+JHtml::_('formbehavior.chosen', 'select');
+
+$app = JFactory::getApplication();
+$input = $app->input;
+$assoc = JLanguageAssociations::isEnabled();
+
+?>
+
```

Comment:
the side by side will probably needs two iframes.
one for the reference content, the other for the associated content

ex:~
```html
	<iframe class="iframe jviewport-height70" src="index.php?option=com_contact&amp;view=contact&amp;layout=modal&amp;tmpl=component&amp;task=contact.edit&amp;id=1" name="Edit Contact" height="400px" width="800px"></iframe>
	<iframe class="iframe jviewport-height70" src="index.php?option=com_contact&amp;view=contact&amp;layout=modal&amp;tmpl=component&amp;task=contact.edit&amp;id=2" name="Edit Contact" height="400px" width="800px"></iframe>
```

of course this will also need some css style to make the iframes render side by side.